libreoffice-core/sw/source/core/text/atrstck.cxx

```diff
@@ -212,16 +212,25 @@ static bool lcl_ChgHyperLinkColor( const SwTextAttr& rAttr,
         OUString sectionUrl = "section://";
         OUString sectionRefUrl = "sectionref://";
         if(aDestinationURL.startsWith(termUrl)){
-            *pColor = COL_PIP_TERM;
+            if(pColor)
```

Comment:
@whutchinson98 Worth monitoring if some items aren't being highlighted because they don't have formatting set (no color applied) or something along those lines. It looks like `RES_CHRATR_COLOR != rItem.Which()` should actually prevent pColor from not being set (because if it has the color attribute, then it should have a color, but these additional checks are a good choice for preventing a crash.